dashboard/test/ui/features/gdpr_dialog.feature

```diff
@@ -0,0 +1,49 @@
+@dashboard_db_access
+
+Feature: GDPR Dialog - data transfer agreement
+
+  Scenario: EU user sees the GDPR Dialog on dashboard, opt out
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then element ".ui-test-gdpr-dialog" is visible
+    Then I click selector ".ui-test-gdpr-dialog-logout"
+    Then check that I am on "http://code.org/"
+    Then I wait until element ".ui-test-gdpr-dialog" is not visible
```

Comment:
It's funny - this test correctly checks redirect behavior, but doesn't directly check whether the user was logged out.  Do you think that's something we should add?  There's a step for that:

https://github.com/code-dot-org/code-dot-org/blob/0d8343074b9f9725e79cd28fdab8c8ba389966e5/dashboard/test/ui/features/step_definitions/steps.rb#L1116-L1118